package.json

```diff
@@ -249,10 +249,10 @@
     "electron-updater": "^4.0.6",
     "electron-window-state": "^5.0.3",
     "find-free-port": "^2.0.0",
-    "get-port": "^4.2.0",
-    "ionicons": "^4.5.5",
+    "get-port": "^5.0.0",
+    "ionicons": "^4.5.6",
     "isomorphic-fetch": "2.2.1",
-    "jquery": "^3.3.1",
+    "jquery": "^3.4.0",
```

Comment:
Looked for it and couldn't find where it's used. Is it still needed ?

Reply:
Sadly, its [required by bootstrap](https://getbootstrap.com/docs/4.3/getting-started/introduction/#js) 😢